common/unsafe/src/main/java/org/apache/spark/unsafe/Platform.java

```diff
@@ -44,6 +44,28 @@
   public static final int DOUBLE_ARRAY_OFFSET;
 
   private static final boolean unaligned;
+  static {
```

Comment:
Ur, #24825 touched `unaligned` initialization and just got merged. Please rebase with the master.

Reply:
@viirya  : are u asked me to rebase with master for Platform.java changes??

Reply:
Yes, otherwise it can't be built.